code/iaas/config-item/common/src/main/java/io/cattle/platform/configitem/version/impl/ConfigItemStatusManagerImpl.java

```diff
@@ -67,7 +59,7 @@
     AgentLocator agentLocator;
```

Comment:
Cant comment on it because its above the fold, but you have two unused private statics at lines 46 and 47 causing warnings.